src/raftstore/store/snap.rs

```diff
@@ -224,6 +241,7 @@ pub struct SnapManagerCore {
     base: String,
     registry: HashMap<SnapKey, Vec<SnapEntry>>,
     ch: Option<SendCh<Msg>>,
+    snap_size: Arc<RwLock<u64>>,
```

Comment:
@c4pt0r says Raspberry PI will support 64 bit later, so it is not the problem later. But now we can use RwLock here.